src/Controller.js

```diff
@@ -107,11 +111,81 @@ class ActivationSingleton {
 		}
 	}
 
+	/**
+	 * Launch WWT application
+	 *
+	 * @return {jQuery.Promise} Promise that is resolved when the system
+	 *  has finished launching, or is rejected if the system has failed to launch
+	 */
+	launch() {
+		if ( !this.model.isEnabled() ) {
+			window.console.log( 'Who Wrote That: Could not launch. System is disabled.' );
+			return $.Deferred().reject();
+		}
+
+		return this.loadDependencies().then( () => {
+			if ( !this.app ) {
+				// Only load after dependencies are loaded
+				// And only load once
+				// We do this trick because our widgets are dependent
+				// on OOUI, which is not available at construction time.
+				// When we launch, we load dependencies and the first run
+				// should make sure we also initialize the widget app
+				const App = require( './App' );
+				this.app = new App();
+			}
+
+			this.model.toggleActive( true );
+			this.model.setState( 'pending' );
+			return this.api.getData( window.location.href )
+				.then(
+					// Success handler.
+					() => {
+						// Insert modified HTML.
+						this.model.cacheOriginal();
+						this.model.getContentWrapper()
+							.html( this.api.getReplacementHtml() );
+						this.model.setState( 'ready' );
+
+						return this.model.getContentWrapper();
+					},
+					// Error handler.
+					errorCode => {
+						this.model.setState( 'err', errorCode );
+					}
+				);
+		} );
+	}
+
+	/**
+	 * Close the WWT application
+	 */
+	dismiss() {
+		if ( !this.model.isActive() ) {
+			window.console.log( 'Who Wrote That: Could not dismiss. System is not active.' );
+			return;
+		}
+		this.model.toggleActive( false );
+		// Revert back to the original content
+		this.model.getContentWrapper()
+			.html( this.model.getOriginalContent().html() );
+	}
+
+	/**
+	 * Toggle the system; if already launched, dismiss it, and vise versa.
+	 */
+	toggle() {
+		if ( this.model.isActive() ) {
+			this.dismiss();
+		} else {
+			this.launch();
+		}
+	}
```

Comment:
New line below this? There is an eslint rule for this I'm sure, if want to use it.

Reply:
Yeah let's add that rule, I'll check what it is. I thought it was already in there, but maybe not for es6 modules rules...?